src/app/forms/services/dynamic-form.types.ts

```diff
@@ -164,6 +164,24 @@ export class CustomFormGroup extends FormGroup implements CustomGroup, BaseForm
   get cleanValueChanges() {
     return this.valueChanges.pipe(map(() => this.getCleanValue(this)));
   }
+
+  // override patchValue(
```

Comment:
This is a draft for something later I think i need to do, please ignore for now

Reply:
oops, ignore my comment to remove it then 😂